app/controllers/api/v1/admin/roles_controller.rb

```diff
@@ -18,6 +18,15 @@ def index
           render_json data: roles
         end
 
+        # GET /api/v1/admin/roles/:role_id.json
+        # Expects: {}
+        # Returns: { data: Array[serializable objects] , errors: Array[String] }
+        # Does: Fetches and returns a role data.
+
+        def show
+          render_json data: @role
```

Comment:
@farhatahmad do I create a `RoleSerializer` and use `render_data` in this PR?
Or you're planning it along with the rest of the actions in a seperate PR?

Reply:
You can create in another PR if you like